docs/contributors/dev-flow.md

```diff
@@ -22,7 +22,6 @@ Build and start the Pipeline Manager (that also serves the Feldera Web Console):
 ```bash
 RUST_LOG=info RUST_BACKTRACE=1 cargo run --bin pipeline-manager --features pg-embed -- --api-server-working-directory ~/.dbsp -d postgres-embed --dev-mode --bind-address 0.0.0.0 --sql-compiler-home ./sql-to-dbsp-compiler --dbsp-override-path .  --compiler-working-directory ~/.dbsp --runner-working-directory ~/.dbsp
 ```
-RUST_LOG=info RUST_BACKTRACE=1 AUTH_CLIENT_ID=44jttbufih62p9hh999f3irjkm AUTH_ISSUER="https://cognito-idp.us-east-1.amazonaws.com/us-east-1_gkyurqf2T" AWS_COGNITO_REGION="us-east-1" AWS_COGNITO_USER_POOL_ID="us-east-1_gkyurqf2T" cargo run --bin pipeline-manager --features pg-embed -- --api-server-working-directory ~/.dbsp -d postgres-embed --dev-mode --bind-address 0.0.0.0 --sql-compiler-home ./sql-to-dbsp-compiler --dbsp-override-path .  --compiler-working-directory ~/.dbsp --runner-working-directory ~/.dbsp --auth-provider=aws-cognito
 
 > Here, `~/.dbsp` is the directory that will host compilation artifacts of SQL Compiler that Pipeline Manager will then use. It will be created if it doesn't exist, and you can use another name for it - just replace corresponding arguments in the above command.
```

Comment:
Not related to this PR, but there's some stale documentation I missed from a previous pass: could you also change `~/.dbsp` to `~/.feldera`?